docs/content/examples/tutorials/crowdsec.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Tutorials | Crowdsec'
+---
+
+!!! quote "What is Crowdsec ?"
+    
+    Crowdsec is an open source software that detects and blocks attackers using log analysis.
+    It has access to a global community-wide IP reputation database.
+
+    [Source][crowdsec]
+
+    [crowdsec]: https://www.crowdsec.net
+
+## Installation
+
+Crowdsec supports multiple [installation methods][crowdsec-installation-docs], however this page will use the docker installation.
+
+
+### Docker mailserver
+
+In the docker-compose definition of the docker-mailserver, create a bind volume containing the logs.
+
+```yaml
+services:
+  mailserver:
+    image: ghcr.io/docker-mailserver/docker-mailserver:latest
+    container_name: mailserver
+    # Provide the FQDN of your mail server here (Your DNS MX record should point to this value)
+    hostname: mail.example.com
+    ports:
+      - "25:25"
+      - "465:465"
+      - "587:587"
+      - "993:993"
+    volumes:
+      - /docker-data/dms/mail-data/:/var/mail/
+      - /docker-data/dms/mail-state/:/var/mail-state/
+      - /docker-data/dms/mail-logs/:/var/log/mail/
+      - /docker-data/dms/config/:/tmp/docker-mailserver/
+      - /etc/localtime:/etc/localtime:ro
+    environment:
+      - ENABLE_RSPAMD=1
+      - ENABLE_CLAMAV=1
+      - ENABLE_FAIL2BAN=0
+    restart: always
+```
+
+### Crowdsec
+
+The crowdsec container needs to have access to the docker-mailserver logs.
+
+```yaml
+services:
+  image: crowdsecurity/crowdsec
+  restart: unless-stopped
+  ports:
+    - "8080:8080"
+    - "6060:6060"
+  volumes:
+    - /docker-data/dms/mail-logs/:/var/log/dms:ro
+    - ./acquis.d:/etc/crowdsec/acquis.d
+    - crowdsec-db:/var/lib/crowdsec/data/
+  environment:
+    COLLECTIONS: crowdsecurity/postfix crowdsecutory/dovecot # These collection contains parsers and scenarios for postfix and dovecot
+    TZ: Europe/Paris
+volumes:
+  crowdsec-db:
+```
+
+## Configuration
+
+Configure crowdsec to read and parse dms logs file.
+
+Create the file `dms.yml` in `./acquis.d/`
+```yaml
+---
+source: file
+filenames:
+  - /var/log/dms/mail.log
+labels:
+  type: syslog
+```
```

Comment:
Uppercase DMS, optionally wrap example in the example admonition.

````suggestion
Configure crowdsec to read and parse the DMS log files.

!!! example

    Create the file `dms.yml` in `./acquis.d/`

    ```yaml
    ---
    source: file
    filenames:
      - /var/log/dms/mail.log
    labels:
      type: syslog
    ```
````